docs/roman/tweakreg/README.rst

```diff
@@ -0,0 +1,293 @@
+  (Default=2.0)
+
+* ``use2dhist``: A boolean indicating whether to use 2D histogram to find
+  initial offset. (Default=True)
+
+* ``separation``: Minimum object separation in arcsec. (Default=1.0)
+
+* ``tolerance``: Matching tolerance for ``xyxymatch`` in arcsec. (Default=0.7)
+
+**Catalog fitting parameters:**
+
+* ``fitgeometry``: A `str` value indicating the type of affine transformation
+  to be considered when fitting catalogs. Allowed values:
+
+  - ``'shift'``: x/y shifts only
+  - ``'rshift'``: rotation and shifts
+  - ``'rscale'``: rotation and scale
+  - ``'general'``: shift, rotation, and scale
+
+  The default value is "rshift".
+
+  .. note::
+      Mathematically, alignment of images observed in different tangent planes
+      requires ``fitgeometry='general'`` in order to fit source catalogs
+      in the different images even if mis-alignment is caused only by a shift
+      or rotation in the tangent plane of one of the images.
+
+      However, under certain circumstances, such as small alignment errors or
+      minimal dithering during observations that keep tangent planes of the
+      images to be aligned almost parallel, then it may be more robust to
+      use a ``fitgeometry`` setting with fewer degrees of freedom such as
+      ``'rshift'``, especially for "ill-conditioned" source catalogs such as
+      catalogs with very few sources, or large errors in source positions, or
+      sources placed along a line or bunched in a corner of the image (not
+      spread across/covering the entire image).
+
+* ``nclip``: A non-negative integer number of clipping iterations
+  to use in the fit. (Default = 3)
+
+* ``sigma``: A positive `float` indicating the clipping limit, in sigma units,
+  used when performing fit. (Default=3.0)
+
+**Absolute Astrometric fitting parameters:**
+
+Parameters used for absolute astrometry to a reference catalog.
+
+* ``abs_refcat``: String indicating what astrometric catalog should be used.
+  Currently supported options are (Default='GAIADR3'):
+
+  - a path to an existing reference catalog (in a supported format);
+
+  -  `None` (see note_ below);
+
+  - empty string (see note_ below);
+
+  - any of the following catalog names (case insensitive) supported by `MAST Catalogs`_:
+
+    + ``'HSC'`` (Hubble Source Catalog);
+
+    + ``'Galex'`` (GALEX Catalog V2 and V3);
+
+    + ``'Gaia'`` (if only the mission name is provided, then DR2 data will be fetched);
+
+    + ``'GaiaDR#'`` (if the mission name is followed by DR1, DR2, or DR3, then data
+      from that specific DR will be fetched);
+
+    + ``'TGAS'`` (Tycho-Gaia Astrometric Solution);
+
+    + ``'TIC'`` (TESS Input Catalog);
+
+    + ``'CTL'`` (TESS Candidate Target List);
+
+    + ``'PanSTARRS'`` (DR1 and DR2);
+
+    + ``'Plato'`` (All-Sky PLATO Input Catalog DR1).
+
+  .. _note:
+  .. note::
+    When ``abs_refcat`` is `None` or an empty string, alignment to the
+    absolute astrometry catalog will be turned off.
```

Comment:
What happens in the L2 case where if abs_refcat is None but there is only one image, so no overlaps?

Reply:
With the latest changes, if `abs_refcat = None`, `TweakRegStep` will use the default catalog as set by `tweakreg.DEFAULT_ABS_REFCAT`.